lib/animated_icon_button.dart

```diff
@@ -141,6 +141,8 @@ class AnimatedIconButton extends StatefulWidget {
   /// background color whenever [AnimatedIconButton] is pressed.
   final AnimationController animationController;
 
+  double splashRadius;
```

Comment:
Please make this field final and.
And enter information about this field in the small documentation.